src/main/java/com/gtnewhorizons/angelica/proxy/ClientProxy.java

```diff
@@ -76,9 +81,23 @@ public void init(FMLInitializationEvent event) {
             FMLCommonHandler.instance().bus().register(Iris.INSTANCE);
             MinecraftForge.EVENT_BUS.register(Iris.INSTANCE);
         }
+
+
+        FMLCommonHandler.instance().bus().register(this);
+        MinecraftForge.EVENT_BUS.register(this);
+
+        glsmKeyBinding  = new KeyBinding("Print GLSM Debug", Keyboard.KEY_GRAVE, "Angelica Keybinds");
```

Comment:
This should probably be set to NONE by default, it's just needless keybind pollution for most players.